docs/integrations/amazon-aws/index.md

```diff
@@ -12,7 +12,325 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 This guide has documentation for all of the apps that Sumo provides for Amazon and AWS. See also: [AWS Observability Solution](/docs/observability/aws).
 
-import DocCardList from '@theme/DocCardList';
-import {useCurrentSidebarCategory} from '@docusaurus/theme-common';
-
-<DocCardList items={useCurrentSidebarCategory().items}/>
+<div className="box-wrapper" markdown="1">
+<div className="box smallbox1 card">
+  <div className="container">
+  <img src={useBaseUrl('/../../../img/integrations/amazon-aws/aws-aurora-logo.png')} alt="Thumbnail icon" width="50"/>
+  <h4><a href="/docs/integrations/amazon-aws/">Amazon Aurora MySQL ULM</a></h4>
```

Comment:
@Melvin-CnC , can you please finish the URLs for this page? 

For example, this URL in this particular line would be /docs/integrations/amazon-aws/**aurora-mysql-ulm**

Reply:
Sure @kimsauce, I'll make the changes. Thought, I'd already made them.